Add information about energy backfilling in waterfurnace (#44654)
home-assistant/home-assistant.github.io: source/_integrations/waterfurnace.markdown

The integration is also able to track historic energy use. You can track the total energy consumption in the energy dashboard. This data is refreshed every 2 hours, so your energy use may lag behind.

> On first setup, the integration will automatically try to fetch the last 13 months of available data. This may take up to 40 minutes to complete.

Reviewer: Let's use the present tense, refactor "automatically" because it doesn't add anything here + use "can" for possibility:

```suggestion
On first setup, the integration tries to fetch the last 13 months of available data. This can take up to 40 minutes to complete.

```
